.github/actions/linux/build-libmongoc/action.yml

```diff
@@ -0,0 +1,55 @@
+name: "Build libmongoc"
+description: "Builds and install libmongoc"
+inputs:
+  prefix:
+    description: "Prefix to install libmongoc to"
+    default: "/usr/local"
+  version:
+    description: "Libmongoc version to build"
```

Comment:
Noted that the description here is missing "(major.minor)", which you had in the build-libmongocrypt action.

Reply:
Ah, yes. Added in #1848